src/PharoThings-Devices-HD44780.package/PotLCD.class/instance/syncClock.st

```diff
@@ -0,0 +1,3 @@
+writing
+syncClock
+	(gpioDic at: #clockPin) value: 0; value: 1; value: 0
```

Comment:
Here for example: 
It can be just a variable clockPin:
```Smalltalk
clockPin value: 0; value: 1; value: 1.
```
Why it is not like this? (and other similar users of gpioDic)

Reply:
Yes, I get it.